libs/ui-toolkit/src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -7,8 +7,8 @@ import { Icon } from '../icon';
 const itemClass = classNames(
   'relative flex items-center justify-between rounded-sm p-2 text-sm',
   'cursor-default hover:cursor-pointer',
-  'hover:bg-white dark:hover:bg-neutral-200',
-  'focus:bg-white dark:focus:bg-neutral-200',
+  'hover:bg-white dark:hover:bg-neutral-500',
```

Comment:
chore (blocking): Can you convert these to use the vega gray scale

Reply:
updated to use vega-light scale and vega-dark scale